### ISSUE TITLE
Dropdown options announced as "Checked" via any browser and JAWS/NVDA screen reader combination.

### ISSUE THREAD

**Issue by escetic** [angular/material#11047]
**Bug, feature request, or proposal:**

Note: This may be a dupe of #10969

We have a form with day, month, year date of birth inputs. The month is a dropdown select, with January, February, March, etc. This is the code snippet (below). Using a screen reader (NVDA or JAWS, any browser), as you down arrow through each option, each month option announces as "January checked", "March checked", etc. , but this is a dropdown, not a checkbox or radio input.
<md-input-container class="md-block md-input-has-placeholder ng-animate md-input-invalid" md-no-float="" data-ng-animate="2">
<md-select tabindex="14" class="phx-perinfo1 ng-pristine md-no-asterisk ng-empty ng-invalid ng-invalid-required ng-valid-maxlength ng-touched" id="select_7" role="listbox" aria-disabled="false" aria-expanded="false" aria-invalid="true" aria-multiselectable="false" aria-required="true" aria-owns="select_container_8" aria-label="Date of Birth Month." name="dobmonth" required="" ng-focus="dmonth=true" md-no-asterisk="" ng-model="vm.profile.dob.month" placeholder="Month" ng-maxlength="2" ng-change="monthSelected(vm.profile.dob.month)">

```
<md-select-value class="md-select-value md-select-placeholder" id="select_value_label_0">
<span>Month</span>
<span class="md-select-icon" aria-hidden="true"></span>
</md-select-value></md-select>

<div aria-live="assertive" ng-focus="dmonth">
<div class="md-input-messages-animation ng-active" aria-hidden="false" aria-live="assertive" ng-show="loanappForm.dobmonth.$touched &amp;&amp; loanappForm.dobmonth.$invalid" ng-messages="loanappForm.dobmonth.$error">
<!-- ngMessage: required -->
<div class="md-input-message-animation ng-binding ng-scope ng-enter ng-enter-active" style="margin-top: 0px; opacity: 1; transition-property: all; transition-duration: 0.3s; transition-timing-function: linear;" ng-message="required">
<img class="error-icon" role="presentation" alt="error" src="/content/images/wrong.png">Error. Month is required
</div>
</div>
</div>

<select name="dobmonth" tabindex="-1" class="md-visually-hidden" aria-hidden="true">
<option class="ng-binding" aria-checked="false" ng-value="monthNumber">
									lapp.months.values.
</option><option aria-checked="false" selected="" ng-value="vm.profile.dob.month"></option>
</select>

</md-input-container>
```

**What is the expected behavior?**

Select dropdowns aren't something which should be announcing as "checked". Options announce as selected (note that none of the options here should be marked as selected). Radio buttons announce as checked.

**What is the current behavior?**

Select dropdown is announcing each option as checked.

**What are the steps to reproduce?**

Sorry, I cannot share the site. I'm hoping someone can see from the code snippet above why the SELECT options are announcing as "checked", as if they were radio buttons.

**What is the use-case or motivation for changing an existing behavior?**

Announcing a dropdown option as "checked" is unexpected behavior and confusing.

**Which versions of Angular, Material, OS, TypeScript, browsers are affected?**

Angular JS Material: v1.1.5.  Assume latest IE, FF, Chrome.

**Is there anything else we should know?**

Thank you in advance.

**Comment by Eradev** [comment]
That's because ngAria adds it to elements with ngValue.

You can add this code after line 999 in select.js:
`if (!selectCtrl.isMultiple)
{
    element[0].removeAttribute('aria-checked');
}`

**Comment by Splaktar** [comment]
Please provide a CodePen demonstration of the issue so that we can debug this with a screen reader and provide a fix.

**Comment by Splaktar** [comment]
Closing as dupe of #10969.
